x-pack/plugins/lens/public/drag_drop/drag_drop.scss

```diff
@@ -73,6 +82,7 @@
   transform: translateY(0);
   transition: transform $euiAnimSpeedFast ease-in-out;
   position: relative;
+  z-index: 1;
```

Comment:
Consider using $euiZLevel variables, or otherwise explaining why we need to provide a specific value here https://github.com/elastic/eui/blob/master/src/global_styling/variables/_z_index.scss